src/platform/sdl2-platform.c

```diff
@@ -138,6 +146,64 @@ static char applyRemaps(char c) {
 }
 
 
+static void speechCallback(short *wav, int numsamples, espeak_EVENT *events) {
+    return;
+}
+
+
+static boolean audioInit() {
+    espeak_SetSynthCallback(speechCallback);
+    return espeak_Initialize(AUDIO_OUTPUT_PLAYBACK, 0, NULL, 0) != -1;
+}
+
+static void _playSpeech(
+    char *text,
+    enum messageFlags flags
+) {
+    if (!useTTS) {
+        return;
+    }
+    long priority = flags & (NO_SPEECH | ZERO_SPEECH | ONE_SPEECH | TWO_SPEECH | THREE_SPEECH);
+    if (lastSpeech.flags & SPEECH_BLOCKS) {
+        espeak_Synchronize();
+    } else if (priority >= lastSpeech.priority) {
+        espeak_Cancel();
+    }
+    // remove any escape codes from the text
+    int i, j;
+    char sanitizedMessage[90] = "";
+    color white;
+    for(i = 0, j = 0; i < strlen(text); i++)
+    {
+        if (text[i] == COLOR_ESCAPE) {
+            i = decodeMessageColor(text, i, &white);
+        }
+
+        if(i >= strlen(text)) {
+            break;
+        }
+
+        sanitizedMessage[j++]= text[i];
+    }
+    printf(sanitizedMessage);
```

Comment:
This printf seems like it will interfere with console interfaces.  Is it just debug logging that can be removed?

Reply:
yep! still just figuring out some bugs. occasionally there's some garbage synthesized that i need to figure out